readthedocs/oauth/utils.py

```diff
@@ -25,6 +18,9 @@
 
 def update_webhook(project, integration, request=None):
```

Comment:
reading the function, we could pass the user (required) and return if the webhook was updated or not, and the function calling this will send the message according to that.